packages/ui/styles/constants/_colors.scss

```diff
@@ -26,3 +26,5 @@ $colorAccessibilityOutline: #9648bd !default;
 $colorBrand: #00bac6 !default;
 $colorShadow: rgb(53 61 73 / 20%) !default;
 $colorMenuDark: #333 !default;
+$colorCustom4: #c6ff3a !default;
+$colorCustom7: #061827 !default;
```

Comment:
It's our new main color for text, so please just redefine `$colorText`

Reply:
done